LineageABCmodel.jl

```diff
@@ -553,20 +553,21 @@ function runlineageABCmodel( lineagetree::Lineagetree, state_init::Lineagestate2
 
     return state_hist, logprob, trickycell
 end     # end of runLineageABCmodel function
-function ABCinitialiseLineageMCmodel( lineagetree::Lineagetree, model::UInt,timeunit::Float64,tempering::String, comment::String,chaincomment::String,timestamp::DateTime, MCstart::UInt,burnin::UInt,MCmax::UInt,subsample::UInt, state_init::Lineagestate2,pars_stps::Array{Float64,1}, nomothersamples::UInt,nomotherburnin::UInt, without::Int64,withwriteoutputtext::Bool )
+
+function ABCinitialiseLineageMCmodel( lineagetree::Lineagetree, model::String, timeunit::Float64,tempering::String, comment::String,chaincomment::String,timestamp::DateTime, MCstart::UInt,burnin::UInt,MCmax::UInt,subsample::UInt, state_init::Lineagestate2,pars_stps::Array{Float64,1}, nomothersamples::UInt,nomotherburnin::UInt, without::Int64,withwriteoutputtext::Bool )
     # initialise parameters
     
     # get parameters:
     nocells = lineagetree.nocells                           # number of cells in data/lineagetree
     (noups, noglobpars,nohide,nolocpars) = getMCmodelnoups2( model, nocells )
     if( size(state_init.pars_glob,1)!=noglobpars )
-        @printf( " (%s) Warning - ABCinitialiseLineageMCmodel (%d): Missmatch of number of global parameteters and initial state (%d vs %d) for model %d.\n", chaincomment,0, noglobpars, size(state_init.pars_glob,1), model )
+        println(" (", chaincomment, ") Warning - ABCinitialiseLineageMCmodel (0): Missmatch of number of global parameteters and initial state (", noglobpars, " vs ", size(state_init.pars_glob,1), ") for model ", model, ".")
     end     # end if incompatible size
 
     # get uppars:
     (statefunctions,targetfunctions, dthdivdistr) = deepcopy( getstateandtargetfunctions( model ) )

```

Comment:
I couldn't find the definition of this `getstateandtargetfunctions` function anywhere.  I take this [code path is dead](https://en.wikipedia.org/wiki/Dead_code) (which is a strong indication that it should just be deleted.